doc/modules/grid_search.rst

```diff
@@ -1,10 +1,10 @@
-.. currentmodule:: sklearn.grid_search
+.. currentmodule:: sklearn.model_selection
 
 .. _grid_search:
 
-===============================================
-Grid Search: Searching for estimator parameters
-===============================================
+==========================================
+Search: Searching for estimator parameters
```

Comment:
> I don't feel the "Search:" adds much.

Indeed... do you suggest we merge this section into model_selection itself (as search is more or less the most important(?) part of model_selection)